test/libyul/yulInterpreterTests/prevrandao.yul

```diff
@@ -0,0 +1,10 @@
+{
+  sstore(0, prevrandao())
+}
+// ====
+// EVMVersion: >=paris
```

Comment:
Another `yulInterpreterTest` for `sstore(0, difficulty())` and `EVMVersion: >=paris`? Don't know if this makes sense for the `yulInterpreterTests` though - as in, should these tests always compile and produce output?

Reply:
`difficulty()` is not anymore a built-in for evm >= paris (this was supposed to be tested in the test that you mentioned above: `inline_assembly_instructions_disallowed_paris.sol`). But it is still available through the magic variable `block` (i.e. `block.difficulty`) for all evm versions (but with the semantic changes).